core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -352,7 +356,7 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
   }
 
   @Test
-  def testConsumeWithNoTopicAccess() {
+  def testConsumeWithoutTopicDescribeAccess() {
```

Comment:
@edoardocomar Hmm... Yeah, that's a fair point. How about we catch the error explicitly and throw as `KafkaException` with a message indicating the topic/partition is unknown which mentions that the the user may not have describe access.